lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -112,10 +115,22 @@ IPV_IDENTITY_REUSE_PAGE:
     next:
       targetState: END
 
+IPV_IDENTITY_PENDING_PAGE_TEST:
```

Comment:
Could you just pop a comment on this to say this is temporary so we can test the F2F reset feature behind a feature flag